client/src/app/site/motions/services/motion-pdf.service.ts

```diff
@@ -71,7 +71,7 @@ export class MotionPdfService {
      *                         this selection will be ignored and all comments exported
      * @returns doc def for the motion
      */
-    public motionToDocDef(motion: ViewMotion, exportInfo?: MotionExportInfo): object {
+    public motionToDocDef(motion: ViewMotion, exportInfo?: MotionExportInfo, flowText: boolean = false): object {
```

Comment:
Perhaps, the name `flowText` is misleading: You use it to en-/disable the use of a title/subtitle. Therefore, name it something like `useTitle` or something else (maybe, this should also be done in the files above).